valhalla/baldr/graphtile.h

```diff
@@ -795,6 +795,28 @@ class GraphTile {
     return traffic_tile;
   }
 
+  /**
+   * Is there an opposing edge with matching edgeinfo offset. The end node of the directed edge
+   * must be in the same tile as the directed edge.  This is called during the building of the
+   * tiles; therefore, we can't use GetOpposingEdgeId as it has not been set yet.
+   * @param  directededge  Directed edge to check.
+   */
+  bool OpposingEdgeInfoDiffers(const DirectedEdge* edge) const {
```

Comment:
can we put this in mjolnir graphtilebuilder or util? in my mind we don thave any methods in the baldr versions of these things that dont work around the tiles being all the way built (ie that validator has fixed up opposing edges). would you be cool with moving into mjolnir somewhere (or maybe you said that originally, sorry again)

Reply:
sure